include/SFML/Graphics/Glyph.hpp

```diff
@@ -52,6 +52,8 @@ class SFML_GRAPHICS_API Glyph
     // Member data
     ////////////////////////////////////////////////////////////
     float     advance;     //!< Offset to move horizontally to the next character
+    int       lsb_delta;   //!< Left offset after forced autohint. Internally used by getKerning()
+    int       rsb_delta;   //!< Right offset after forced autohint. Internally used by getKerning()
```

Comment:
These two members don't follow our naming convention (should be `lsbDelta` and `rsbDelta`).